test/data/tsang-falloff.yaml

```diff
@@ -0,0 +1,55 @@
+generator: cti2yaml
+cantera-version: 2.6.0a2
+date: Mon, 30 Aug 2021 16:33:51 +0300
+input-files: [Lucassen-2011.cti]
+description:
+  Sample reactions in the Tsang falloff format for testing. Taken from Lucassen et al. 2011 (https://doi.org/10.1016/j.combustflame.2011.02.010)
```

Comment:
Whenever there's a description, it usually starts on line 1 (see, e.g. `h2o2.yaml` [here](https://github.com/Cantera/cantera/blob/main/data/h2o2.yaml)), with an empty line separating it from the next block. Also, try to limit line length to 88 characters.

Reply:
Fixed and PR updated